environments.json

```diff
@@ -1284,14 +1284,14 @@
     "family": "V8",
     "short": "CH&nbsp;59,<br>OP&nbsp;46",
     "note_id": "experimental-flag",
-    "obsolete": false
+    "obsolete": true
```

Comment:
That's been the strategy for all browsers that are on the rolling-release schedule (Edge is an exception as it's not really that rapid). Within a few days most people will get an upgrade and will get it applied when they restart the browser.

Giving up on this approach would just create the need for lots of little arbitrary decisions on when to flip the switch which would happen soon enough anyway.